WinUIGallery/ControlPages/DesignGuidance/IconsPage.xaml

```diff
@@ -101,45 +102,67 @@
-                    .<LineBreak />
-                </Paragraph>
-            </RichTextBlock>
+        <TextBlock Text="With the release of Windows 11, Segoe Fluent Icons Font is the recommended symbol icon font."/>
+        <labs:SettingsExpander Grid.Row="1"
+                  IsExpanded="False"
+                  Header="Instructions on how to use Segoe Fluent Icons">
+            <labs:SettingsExpander.Items>
+                <labs:SettingsCard ContentAlignment="Left" Padding="16,8">
+                    <RichTextBlock TextWrapping="Wrap">
+                        <Paragraph FontWeight="SemiBold">
+                            How to get the font
+                        </Paragraph>
+                        <Paragraph>
+                            On Windows 11: There's nothing you need to do, the font comes with Windows.<LineBreak />
+                            On Windows 10: The Segoe Fluent Icons is not included by default on Windows 10. You can download it
+                            <Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/downloads/#fonts">here</Hyperlink>.
+                        </Paragraph>
+                    </RichTextBlock>
+                </labs:SettingsCard>
+                <labs:SettingsCard ContentAlignment="Left" Padding="16,8">
+                    <StackPanel Orientation="Vertical" Spacing="8">
+                        <RichTextBlock TextWrapping="Wrap">
+                            <Paragraph FontWeight="SemiBold">
+                                How to use the font
+                            </Paragraph>
+                            <Paragraph>
+                                If you don't specify a <Run FontFamily="Consolas, monospace" FontWeight="Bold">FontFamily</Run>,
+                                or you specify a <Run FontFamily="Consolas, monospace" FontWeight="Bold">FontFamily</Run> that is not available on the system at runtime,
+                                the <Hyperlink NavigateUri="https://learn.microsoft.com/uwp/api/windows.ui.xaml.controls.fonticon">FontIcon</Hyperlink> falls back to the default font family defined by the
+                                <Run FontFamily="Consolas, monospace" FontWeight="Bold">SymbolThemeFontFamily</Run> theme resource.
+                            </Paragraph>
+                        </RichTextBlock>
+                        <controls:SampleCodePresenter
+                            SampleType="XAML"
+                            CodeSourceFile="Icons/FontIconSample1_xaml.txt">
+                        </controls:SampleCodePresenter>
+                    </StackPanel>
+                </labs:SettingsCard>
+            </labs:SettingsExpander.Items>
+        </labs:SettingsExpander>
 
+        <StackPanel Spacing="8" Grid.Row="2">
```

Comment:
```suggestion
        <StackPanel Spacing="8" Grid.Row="2" Margin="0,24,0,0">
```

We want to have a little bit of visual breathing space between the instructions and the header.

Before:
![image](https://github.com/microsoft/WinUI-Gallery/assets/9866362/f74625a3-29e1-4e4f-b78a-01132bd80d32)

After:
![image](https://github.com/microsoft/WinUI-Gallery/assets/9866362/7e7fb612-4bf0-46a9-997e-b63956f692eb)

Similar to the typography page;
![image](https://github.com/microsoft/WinUI-Gallery/assets/9866362/3bf70ae7-528e-43ae-9c8d-4ebc7e9d33d3)